### DISCUSSION TITLE
RNFB set DB firebase

### DISCUSSION THREAD

**Discussion by vaiosyndicates** [invertase/react-native-firebase#5156] [Q&A]
basically i want to implement RNFB Realtime DB to my app
from your documentation, we just import the package and use database() func
but when i try to log the database(), it got null
so my question is, any configuration needed for define the db url?
thank you

**Comment by mikehardy** [comment]
My best suggestion is to review our e2e test app which lives in `tests/` here along with `packages/database/e2e` to see how it is used on combination with the docs.
